dev/com.ibm.ws.microprofile.reactive.messaging_fat/fat/src/com/ibm/ws/microprofile/reactive/messaging/fat/suite/SaslPlainTests.java

```diff
@@ -26,6 +28,8 @@
  */
 @RunWith(Suite.class)
 @SuiteClasses({ KafkaSaslPlainTest.class,
+                LibertyLoginModuleTest.class,
+                LibertyLoginModuleXorTest.class,
```

Comment:
Do we also need tests for 1) no crypto algorithm 2) invalid algorithm 3) badly encoded password ?

Reply:
Yeah, those are good tests. I'll have to do a bunch of test refactoring to avoid massive amounts of duplication.

Reply:
Done